osu.Game/Online/Spectator/FrameHeader.cs

```diff
@@ -15,57 +15,75 @@ namespace osu.Game.Online.Spectator
     public class FrameHeader
     {
         /// <summary>
-        /// The current accuracy of the score.
+        /// The total score.
         /// </summary>
         [Key(0)]
+        public long TotalScore { get; set; }
```

Comment:
As long as we're happy with this being a breaking change (I guess it's fine).

Reply:
As a note, I'm 95% sure that _any_ change to a MessagePack model is a guaranteed breaking change, and that adding members to the end doesn't prevent it.

Reply:
Yeah I think you're right.